modules/ROOT/pages/view-clear-logs-task.adoc

```diff
@@ -1,26 +1,76 @@
+17:36:32.391     03/12/2019     Deployment     system     SYSTEM
+Starting CloudHub worker at 54.172.105.253 ...
+17:37:18.402     03/12/2019     Deployment     system     SYSTEM
+Worker(54.172.105.253): Starting your application on mule=4.1.5...
+17:37:21.147     03/12/2019     Deployment     system     SYSTEM
+Worker(54.172.105.253): Your application has started successfully.
+17:37:21.520     03/12/2019     Deployment     system     SYSTEM
+Your application is started.
+....
+
+Messages from the worker that is assigned to a project:: These are messages that show the different tasks that the worker performs. Here are some example messages.
++
+....
+17:37:18.814     03/12/2019     Worker-0     qtp630288493-49     INFO
 
-Logs, if there are any, appear.
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
++ New app 'thisApp-firs'                    +
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+17:37:18.938     03/12/2019     Worker-0     qtp630288493-49     INFO
+
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
++ Initializing app 'thisApp-firs'           +
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+17:37:18.947     03/12/2019     Worker-0     qtp630288493-49     INFO
+The PersistentQueueManager is NOT configured. The normal VM queue manager will be used.
+17:37:20.397     03/12/2019     Worker-0     qtp630288493-49     INFO
+Registering extension ee (version: 4.1.5 vendor: MuleSoft, Inc. )
+17:37:20.400     03/12/2019     Worker-0     qtp630288493-49     INFO
+Registering extension mule (version: 4.1.5 vendor: MuleSoft, Inc. )
+17:37:20.648     03/12/2019     Worker-0     qtp630288493-49     INFO
+Initializing RegistryBroker
+....
+
+The log file also shows messages from any Logger cards that you've added to flows in your project.
+
+== Procedure
+
+* In the lower-right corner of the canvas, click *Logs*.
++
+image::log-view.png[]
 
-image::to-debug-a-flow-designer-project-30b5a.png[]
+== Result
+The log pane expands and displays the log file.
 
-Alternatively, in Design Center:
+image::log-file.png[]
 
-. Click image:manage-connector-versions-design-center-ced9a.png[] Options.
-. Select Open Logs.
+== What to do next
 
+* If you want to delete the content of the log file, click the dots to the right of the *Logs* label and select *Clear Logs*.
 
-////
-. Select Options > Clear Logs.
+* To view the log file in Runtime Manager, click the dots to the right of the *Logs* label and select *View in Runtime Manager*.
 +
-image::log-options.png[]
+[NOTE]
+====
+If you have not specified a default environment for your profile in Anypoint Platform, you're asked to specify one. After you do that, return to the flow-design canvas and select *View in Runtime Manager* again.
+====
 +
-The log window empties.
-////
+In Runtime Manager, you can view messages in the System log separately from the messages for the worker that is assigned to your project.
```

Comment:
@rddheath i think one section that is missing here is educating the user on how they can 'copy a log line'. This can be done by a user scrolling through the log lines inside the log dialog. On the left hand side they will see a label/button show up that says 'COPY' with a copy icon next to it. Clicking that label/button results in that log line to be copied to the clipboard. This is helpful for users who are looking to export log lines to other applications, or collaborate around specific logger output with coworkers.

Reply:
Done.

Reply:
Well, for some reason, my addition isn't appearing in the review here. This is the line that I added before the bullet point about deleting the log:

* You can copy a line of the log by clicking the *Copy* option to its left.

Reply:
See https://github.com/mulesoft/docs-design-center/blob/DOCS-4339-FD-1-9-0-rh/modules/ROOT/pages/view-clear-logs-task.adoc#what-to-do-next for the change.